### ISSUE TITLE
Continuous mode briefly flashes out of order page 

### ISSUE THREAD

**Issue by krispykiq** [Dudemanguy/mpv-manga-reader#12]
After you reach the end of the rendered pages it flashes an image and immediately goes to normal. It makes it confusing because the image it flashes is out of order.

For example in a file with 20 images, when you reach the end of the page 8, it flashes page 16 then shows page 9. When you reach page 16, it flashes page 20 then goes to normal.

**Comment by Dudemanguy** [comment]
Yeah that's because of how the logic works with observing the pan value and flipping pages. I'll probably refactor this to be better.

**Comment by Dudemanguy** [comment]
I can add some workarounds to make it a little better, but I think you will always have a slight flash because of how this works in mpv. Basically, switching files is an async operation not a blocking one so the pan change happens while it is changing pages. Related mpv issue. https://github.com/mpv-player/mpv/issues/7293

**Comment by krispykiq** [comment]
Why not add the workaround to make it better then show a black rectangle to cover the flicker?

**Comment by Dudemanguy** [comment]
The said workarounds I was thinking of was basically just making the image temporarily black while switching pages. The flicker appears to always be visible though.

**Comment by Dudemanguy** [comment]
Apologies for the delay. I got around to trying to see if I could make this any better (basically turning down brightness/contrast all the way down before changing page), but sadly I do not think there is a real benefit. What will happen is that the page does indeed turn black, but changing pages flickers no matter what. I think this is just how mpv internally works. Unsetting brightness/contrast settings always seems to happen at the same time the vertical alignment is changed (the source of the flicker you are seeing). It doesn't matter if I do something stupid like sleep after changing a page; mpv always works this way. So sadly this isn't fixable at this level.
